airgun/widgets.py

```diff
@@ -1969,6 +1969,17 @@ def _read_all(self):
         return table_rows
 
 
+class SatHostsTable(SatTable):
```

Comment:
may be good to change the class name to something common as this is just not related to hosts. ?

Reply:
The problem with duplicate hidden columns is present only in the Hosts table. Other tables are not affected.
So I created a separate widget in order to isolate it and potentially to not break other tables.

Reply:
The fix is quite general, would it be better to explicitly specify `not hidden` also for other entities, not just host table?

Reply:
Ok, fair point, I've moved the changes to the `SatTable` widget and got rid of the `SatHostsTable` one. Let's hope it won't break any other tables.